packages/web-components/src/menu-item/menu-item.ts

```diff
@@ -111,7 +111,7 @@ export class MenuItem extends FASTElement {
     toggleState(this.elementInternals, 'checked', checkableMenuItem ? next : false);
 
     if (this.$fastController.isConnected) {
-      this.$emit('change');
+      this.$emit('change', next, { bubbles: true });
```

Comment:
Two things - yes on bubbling, soft no on `detail`.

Couldn't this be handled by inspecting the event's `target`? The built-in `change` event emitted by inputs doesn't provide a `detail`, so I worry that this may become something we'll have to support in the future.

Reply:
Open to ideas here but this is the same pattern we use in radio and radio group. To skate around this issue where I noticed the bug I had to use `hasAttribute("checked")` and that proved to be some-what unreliable. `event.detail` guarantees we get the current state when the event is emitted.

Reply:
Considering this will always be a custom event and `this.$emit` requires the `detail` property to be set, this is okay. In the tests, I recommend checking the target's `checked` property - I added suggestions to illustrate that.